qgreenland/util/luigi/tasks/main.py

```diff
@@ -18,7 +18,8 @@
     get_layer_release_dir,
 )
 from qgreenland.util.luigi.target import temporary_path_dir
-from qgreenland.util.provenance import steps_to_provenance_text
+from qgreenland.util.provenance import write_provenance_file
+from qgreenland.util.qgis.metadata import write_metadata_file
```

Comment:
Maybe we should move this module up a level? Most of this just deals with formatting strings for layer metadata/abstract, with a couple of functions that operate on the qgis project level.

Reply:
I think that makes sense :+1:

Reply:
679a8fc9